.github/workflows/linux-workflow.yml

```diff
@@ -1,4 +1,4 @@
-name: Linux Build
+name: 🐧 Linux Builds
```

Comment:
You didn't rebase correctly? Those are master commits

Reply:
Those changes were added in a separate PR 
 https://github.com/PCSX2/pcsx2/pull/4522 after the initial rebase, so added manually after.
https://github.com/PCSX2/pcsx2/pull/4529 will also be touching the same code.

Reply:
Sure, but this still means the rebase isn't correct, those are now part of the master branch

You should

git checkout master
git pull origin master
git checkout app-test
git rebase master